firehose-source/src/test/java/demo/FirehoseApplicationTests.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright 2015 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package demo;
+
+import demo.support.WebSocketConfig;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.WebIntegrationTest;
+import org.springframework.cloud.stream.annotation.Output;
+import org.springframework.cloud.stream.annotation.Source;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+/**
+ * @author Vinicius Carvalho
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = {FirehoseApplication.class, WebSocketConfig.class})
+@WebIntegrationTest
+@DirtiesContext
+public class FirehoseApplicationTests {
+
+    @Autowired
+    @Output(Source.OUTPUT)
+    private MessageChannel output;
+
+    @Test
+    public void contextLoads() {
+
+    }
+
+    @ServiceActivator(inputChannel = Source.OUTPUT)
+    public void receive(Message message) {
+        System.out.println(message.getPayload());
```

Comment:
This will need to change.
There is now support for more meaningful tests in `s-c-s`, see _e.g._ https://github.com/spring-cloud/spring-cloud-stream/blob/3dac86a18acde6e5a489e4f07ad40c7ab1645489/spring-cloud-stream-test-support/src/test/java/org/springframework/cloud/stream/test/ExampleTest.java